app/bundles/ReportBundle/Scheduler/Command/ExportSchedulerCommand.php

```diff
@@ -41,6 +50,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return Command::SUCCESS;
         }
 
+        if (!$this->checkRunStatus($input, $output, $exportOption->getReportId())) {
+            return Command::FAILURE;
```

Comment:
If script running, It should still return success.  Probably unit tests will work then